exchange/exchange-ps/exchange/mailboxes/Get-RecoverableItems.md

```diff
@@ -57,7 +57,7 @@ This example returns all of the available recoverable deleted messages with the
 ## PARAMETERS
 
 ### -Identity
-The Identity parameter specifies the mailbox that contains the Recoverable Items folder that you want to view. You can use any value that uniquely identifies the mailbox. For example:
+The Identity parameter specifies the list of mailbox that contains the Recoverable Items folder that you want to view. You can use any value that uniquely identifies the mailbox. For example:
```

Comment:
The Identity parameter specifies the list of mailboxes that contain the Recoverable Items folder that you want to view. You can use any value that uniquely identify mailboxes. For example: